view/src/main/java/cmm/apps/esmorga/view/eventdetails/EventDetailsViewModel.kt

```diff
@@ -85,7 +85,7 @@ class EventDetailsViewModel(
             }.onFailure { error ->
                 _uiState.value = _uiState.value.copy(primaryButtonLoading = false)
                 if (error.code == ErrorCodes.NO_CONNECTION) {
-                    _effect.tryEmit(EventDetailsEffect.ShowNoNetworkSnackbar)
+                    _effect.tryEmit(EventDetailsEffect.ShowNoNetworkScreenError())
```

Comment:
an this is why I think is a better idea to use a more generic name for effects. `ShowNoNetworkError` would work in both cases and we wouldn't need to rename when moving from a snackbar to a screen